dbus-opendtu.py

```diff
@@ -134,7 +132,7 @@ def _v(_p, value: float) -> str:
                     )
 
         for actual_template in range(number_of_templates):
-            logging.info("Registering Templates")
+            logging.critical("Registering Templates")
```

Comment:
same

Reply:
agree, but it shows at a minimum level und only at startup, that the service is started and what is registered, and only at startup